.travis.yml

```diff
@@ -0,0 +1,11 @@
+---
+language: ruby
+bundler_args: --without development system_tests
+before_install: rm Gemfile.lock || true
+rvm:
+  - 1.8.7
+  - 1.9.3
+  - 2.0.0
+script: bundle exec rake test
+env:
+  - PUPPET_VERSION="3.8.0" STRICT_VARIABLES=yes FUTURE_PARSER=yes STRINGIFY_FACTS=false
```

Comment:
@justinstoller How should things like STRICT_VARIABLES and FUTURE_PARSER be handled in Jenkins config? Just add them?